ibm/mas_devops/roles/suite_app_config/README.md

```diff
@@ -180,6 +180,11 @@ Optional. Flag indicating if manage demodata should be loaded or not.
 - Environment Variable: `MAS_APP_SETTINGS_DEMODATA`
 - Default: `false` (do not load demodata)
 
+### "{{ mas_app_settings_db2vargraphic | bool }}"
```

Comment:
This should just be `mas_app_settings_db2vargraphic`

Reply:
fixed